pkg/providers/instance/instance.go

```diff
@@ -267,6 +255,28 @@ func (p *DefaultProvider) launchInstance(ctx context.Context, nodeClass *v1.EC2N
 	return createFleetOutput.Instances[0], nil
 }
 
+func (p *DefaultProvider) GetCreateFleetInput(nodeClass *v1.EC2NodeClass, capacityType string, tags map[string]string, launchTemplateConfigs []ec2types.FleetLaunchTemplateConfigRequest, dryrun ...bool) *ec2.CreateFleetInput {
```

Comment:
Can we drop the `dryrun` parameter in favor for adding it in the validation controller?